src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -139,6 +139,8 @@ ProblemManager::ProblemManager( conduit::Node & root ):
     setRestartFlags( RestartFlags::WRITE ).
     setDescription( "Whether to disallow using pinned memory allocations for MPI communication buffers." );
 
+  chai::ArrayManager::getInstance()->disableCallbacks();
```

Comment:
why is this needed?

Reply:
I wanted this off by default for all the tests.

Reply:
I'm sure you're right as you mentioned in the other comment that it can be re-enabled if desired.